infomaniak-build-tools/windows/Readme.md

```diff
@@ -143,7 +143,7 @@ F:
 cd Projects
 git clone -b master https://github.com/pocoproject/poco.git
 cd poco
-git checkout poco-1.12.5
+git checkout tags/poco-1.12.5-release
```

Comment:
Is there an issue with the version `1.13.2` on Windows?

Reply:
Yes, I described it [here](https://infomaniak.atlassian.net/browse/KDESKTOP-673?atlOrigin=eyJpIjoiMjI2NTBkNDljMzI3NGYwMDk5NGFhOGZkMWNjOTM3MWMiLCJwIjoiaiJ9)
The XML parser seems to crash on Windows and I could not understand why. So I choose to remain in version 1.12.5 for Windows for the moment

Reply:
I would be nice to leave comment so that users/developers are not tempted to upgrade.

Reply:
Agreed, its done